docs/source/guide/label_object.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Retrieve imported data in the labeling configuration
+short: Retrieve data
+tier: all
+type: guide
+order: 208
+order_enterprise: 108
+meta_title: Retrieve data using the object tag
+meta_description: How to retrieve imported data using object tags.
+parent: "setup"
+parent_enterprise: "setup"
+date: 2024-03-24 02:02:03
+---
+
+
+In order to get data into tasks, you have to [import it](tasks). Then you have to specify how you want to retrieve that data to use it in a task. This is done with object tags. 
```

Comment:
This section and text are very blurry. Let's rewrite it to something like this: 

```suggestion
All object tags are intended to load and display data into the labeling interface. The each object tag has the `value` field and it should point to a `task['data']` dictionary field.
```

Reply:
Then we need to cut the whole Variables section from this page and move it here:
![image](https://github.com/HumanSignal/label-studio/assets/501780/393d8646-1866-472c-a11b-e9a21a1fcc94)

In the original page we should keep Variables section, but make a link to this page.